x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/resolver.tsx

```diff
@@ -33,4 +33,6 @@ export const AlertDetailResolver = styled(
   width: 100%;
   display: flex;
   flex-grow: 1;
+  // ugly demo hack until can play nice with eui
```

Comment:
If this is still here by the time we merge, let's remove the comment and add a github issue instead